src/templates/app/config/webpack/dev.js

```diff
@@ -0,0 +1,6 @@
+const merge = require('webpack-merge');
+const common = require('./common.js');
```

Comment:
We'll probably want to coordinate this with crystal environments as well. Maybe call them development and production unless you think we want to shorten them for the whole project.

Reply:
good call.  renamed to `development` and `production`